packages/terra-profile-image/tests/wdio/profile-image-spec.js

```diff
@@ -0,0 +1,12 @@
+/* global browser, Terra, before */
+
+describe('Profile Image', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/profile-image/profile-image-default'));
+
+    Terra.should.beAccessible();
```

Comment:
can we break this into two snapshots, one for default and one for error?

Reply:
[updated](https://github.com/cerner/terra-core/pull/1500/commits/d6fcb7b7f69779567facda2b3748da26150bc30f).